code/modules/mob/living/basic/vermin/cockroach.dm

```diff
@@ -144,3 +144,58 @@
 
 /datum/ai_behavior/basic_melee_attack/hauberoach //Slightly slower, as this is being made in feature freeze ;)
 	action_cooldown = 1 SECONDS
+
+/mob/living/basic/cockroach/sewer
+	name = "sewer cockroach"
+	desc = "A roach not to be underestimated."
+	icon_state = "cockroach_sewer"
+	health = 2
+	maxHealth = 2
+	melee_damage_lower = 2
+	melee_damage_upper = 4
+	obj_damage = 5
+	gold_core_spawnable = NO_SPAWN
+	ai_controller = /datum/ai_controller/basic_controller/cockroach/sewer
+
+/datum/ai_controller/basic_controller/cockroach/sewer
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/random_speech/cockroach,
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree/sewer,
+		/datum/ai_planning_subtree/find_and_hunt_target/roach,
+	)
+
+/datum/ai_planning_subtree/basic_melee_attack_subtree/sewer
+	melee_attack_behavior = /datum/ai_behavior/basic_melee_attack/sewer
+
+/datum/ai_behavior/basic_melee_attack/sewer
+	action_cooldown = 0.8 SECONDS
+
+/mob/living/basic/cockroach/sewer/glockroach
+	name = "sewer glockroach"
+	icon_state = "glockroach_sewer"
+	melee_damage_lower = 3
+	melee_damage_upper = 12
+	obj_damage = 10
+	gold_core_spawnable = NO_SPAWN
+	ai_controller = /datum/ai_controller/basic_controller/cockroach/glockroach
+	cockroach_cell_line = CELL_LINE_TABLE_GLOCKROACH
+
+/mob/living/basic/cockroach/sewer/glockroach/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/ranged_attacks, /obj/item/ammo_casing/glockroach)
+
+/mob/living/basic/cockroach/sewer/hauberoach
```

Comment:
please do not make sewer subtypes of every maint mob, this is a really bad pattern and you need to find a better way of doing this